src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -26,9 +26,50 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     assignSymbolStoreEntry(fullyQualifiedName, value, input = true)
   }
 
+  for {
+    workflow <- binding.workflows
+    call <- workflow.calls
+    (k, v) <- call.inputMappings
+  } yield assignSymbolStoreEntry(s"${call.fullyQualifiedName}.$k", v, input = true)
+
+  private def callOutputEntries(call: Call): Seq[SymbolStoreEntry] =
+    store.filter {entry => entry.scope == call.fullyQualifiedName && entry.isOutput}.toSeq
+
+  private def callInputEntries(call: Call): Seq[SymbolStoreEntry] =
+    store.filter {entry => entry.scope == call.fullyQualifiedName && entry.isInput}.toSeq
+
```

Comment:
Also these two funcs are nearly identical. There could be a function which does everything but the isOutput/isInput and these two call that w/ the appropriate thing